app/routes/circulars.$circularId.($version)/rehypeAutolinkLiteral.ts

```diff
@@ -20,7 +20,11 @@ const regexp = new RegExp(
 function autolinkLiteral(tree: Parameters<typeof findAndReplace>[0]) {
   findAndReplace(
     tree,
-    [regexp, (href: string) => h('a', { rel: 'external', href }, href)],
+    [
+      regexp,
+      (href: string) =>
+        h('a', { rel: 'external', href, target: '_blank' }, href),
```

Comment:
```suggestion
        h('a', { rel: 'external', href, target: '_blank', rel='noreferrer' }, href),
```

Reply:
tweaking suggestion in its own commit to prevent property from being repeated